player/scripting.c

```diff
@@ -242,14 +242,18 @@ static void load_builtin_script(struct MPContext *mpctx, int slot, bool enable,
 {
     mp_assert(slot < MP_ARRAY_SIZE(mpctx->builtin_script_ids));
     int64_t *pid = &mpctx->builtin_script_ids[slot];
-    if (*pid > 0 && !mp_client_id_exists(mpctx, *pid))
+    if (*pid > 0 && !mp_client_id_exists(mpctx, *pid)) {
+        MP_DBG(mpctx, "Assuming script %s has exited (client dead)\n", fname);
```

Comment:
```suggestion
        MP_DBG(mpctx, "Client %d/%s is no longer alive. Marking as unloaded.\n", fname);
```

Reply:
I assume `%d` would be the pid. it's not logged anywhere else AFAIK so I decided to leave it out of the message